Objects/Converters/ConverterRevit/ConverterRevitShared/RevitCommitObjectBuilder.cs

```diff
@@ -211,7 +216,7 @@ private static string GetCategoryId(Base conversionResult, Element revitElement)
 #endif
```

Comment:
Could not discover where the #if began :O

Reply:
line 212